internal/errors/error.go

```diff
@@ -32,13 +32,13 @@ type Err struct {
 	Wrapped error
```

Comment:
The formatting of this comment doesn't quite render correctly as a list for the supported options.  Happy to help with suggested fixes, but it's not very readable at the moment when rendered as godoc.